pkg/suggestion/nasrl_service.py

```diff
@@ -125,11 +125,6 @@ def GetSuggestions(self, request, context):
                         time.sleep(20)
                         result = self.GetEvaluationResult(request.study_id)
 
-                    # This lstm cell is designed to maximize the metrics
-                    # However, if the user want to minimize the metrics, we can take the negative of the result
-                    if self.opt_direction == api_pb2.MINIMIZE:
-                        result = -result
-
```

Comment:
why delete above?

Reply:
Sorry, I made a mistake. Should delete the code above that. Already fixed

Reply:
ok, thanks